src/main/java/org/mule/extension/db/internal/util/StoredProcedureUtils.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.db.internal.util;
+
+import static java.lang.String.format;
+
+import java.sql.SQLException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Utility class for stored helpful for stored procedure operations.
+ *
+ * @since 1.5.0
```

Comment:
this PR will be cherry-picked to 1.4.x so change this to the proper 1.4.x version

Reply:
ok